testnets/wardenprotocoltestnet2/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "wardenprotocoltestnet2",
```

Comment:
Just double checking on the desired chain name; typically archiving a network iterates a counter to 1 first, instead of 2. Are we sure we don't want to start with archiving as chain_name: 'wardenprotocoltestnet1'?
Whichever it will be, the new assetlist must also have it's chain_name property updated, and so far it was missed.

Reply:
Hey @JeremyParish69 ,
Yes I think you are right, I've now removed Alfama since it's not active anymore.
We will start following the numbering convention from here onwards, e.g. next one will have the "2" in it.
Thank you.